### ISSUE TITLE
[Jasmine + Protractor]More than one launch result appears on ReportPortal when specs are run in parallel

### ISSUE THREAD

**Issue by izabielin** [reportportal/agent-js-jasmine#8]
When we specify more then one spec in protractor config file and trigger test run with chrome capability option shardTestFiles = true (it means, we want to run tests in parallel) - on ReportPortal appears the same count of result launches as number of specs.

**Comment by zabielin** [comment]
hello all, any updates regarding this issue?

**Comment by evjlobanova** [comment]
hello, @zabielin ! could you please try to use Java Script agent that is supporting a multi-threading  https://www.npmjs.com/package/reportportal-agent-jasmine ? Does it work for you?

**Comment by DzmitryHumianiuk** [comment]
@izabielin hi, did it help?

**Comment by evjlobanova** [comment]
@izabielin ^^

**Comment by DzmitryHumianiuk** [comment]
@izabielin @zabielin 
closed without response.
plz submit new issue if still actual
